### ISSUE TITLE
check_periodic should waiting while as less one pair in progress

### ISSUE THREAD

**Issue by marakew** [aiortc/aioice#39]
while test i saw when check_periodic run check_start while prevision paring still in progress
this can produce incorrect nomination

fix like this

```
    def check_periodic(self) -> bool:
 +       for pair in self._check_list:
 +          if pair.state == CandidatePair.State.IN_PROGRESS:
 +              return True

        # find the highest-priority pair that is in the waiting state
```

**Comment by jlaine** [comment]
I don't think this is correct. According to the RFC, the periodic timer triggers new checks whenever it fires, I don't see anything saying the checks must be performed one after another. This would make the process very slow when you have many candidate pairs to test.

Am I reading the RFC wrong?

**Comment by jlaine** [comment]
Hello?

**Comment by marakew** [comment]
I was test on windows with 4 candidates
rfc didnt precise about this case
but rfc says about in order send test

aioice uses logic from pjproject ice, where have case in periodic when candidate in progress, just resend transaction

In many implementation of ice, periodic timer related with transaction stun timer

but it will be interesting ask to authors of ice rfc

**Comment by jlaine** [comment]
I believe this was addresses in 54a575139b398a5d168512bc8c26b13d868068ce
